src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -307,28 +308,45 @@ class SlickDataAccess(databaseConfig: Config, val dataAccess: DataAccessComponen
     runTransaction(action)
   }
 
-  override def getWorkflowsByState(states: Traversable[WorkflowState]): Future[Traversable[WorkflowInfo]] = {
+  override def getWorkflow(workflowId: WorkflowId): Future[WorkflowDescriptor] = {
+    val action = for {
+      workflowExecutionResult <- dataAccess.workflowExecutionsByWorkflowExecutionUuid(workflowId.toString).result.head
```

Comment:
What I'm suggesting with this and the get comments are that we should be getting in the habit of documenting every single one when we do it (not retroactively). 

Is it done because there's an assumption that it must be safe? If so, what is that assumption. Is it done out of laziness and we're fine w/ an exception happening?

This way if/when a None.get happens and someone goes to investigate we know if the problem is that it was never intended to work perfectly or if some upstream assumption has been changed.